### ISSUE TITLE
Keyboard entry does not work with NVDA/JAWS enabled on non-touch devices

### ISSUE THREAD

**Issue by Leon-Lj** [arnog/mathlive#1832]
## Description
On non touch devices (such as a desktop or a laptop without a touch screen), if you enable a screen reader, NVDA or JAWS, input from the (physical) keyboard will not work. Attempting to type anything into the math field will just result in the screen reader trying to use it's hotkeys (and usually failing). It does not understand that it has focus in an input field and thus does not release its hotkeys. It doesn't matter if you focus the math field with tab or click, neither works.
On touch devices (such as a Windows laptop with a touch screen) it works just fine.

If I modify the function `export function isTouchCapable(): boolean` to always return true, then everything works just fine.

## Steps to Reproduce

1. On a non touch device (such as a desktop or laptop without touch screen)
2. Start either NVDA or JAWS
3. Go to https://cortexjs.io/mathlive/demo/
4. Focus on the math field by either tabbing to it, or by clicking on it
5. Press 2 on the keyboard

### Actual Behavior

The screen reader reads "No next headlines on level 2"

### Expected Behavior

The number 2 should be added to the math field.

### Environment

**MathLive version**
0.89.1

**Operating System**
Windows 10, 21H2 (OS Build 19044.2486)

**Browser**
Chrome 109.0.5414.120
Firefox 110.0 (64-bit)

**Screen readers**
NVDA - 2022.4 (2022.4.0.27401)
JAWS - 2023.2302.15

**Comment by Leon-Lj** [comment]
This can seemingly be fixed in two ways by modifying the row
```
    markup += `<textarea class=ML__textarea__textarea autocapitalize=off autocomplete=off autocorrect=off spellcheck=false inputmode=none aria-hidden=true tabindex=${
      element.tabIndex ?? 0
    }></textarea>`;
```

Setting `aria-hidden=false` solves it, or setting `tabindex=0` solves it. It also works with setting both of them, which I would guess is the most correct? An input box that's "visible" should be tab-able, and vise versa?

But not matter how I tinker with the textarea I can't get the accessibility tree to stop claiming that it breaks WCAG.
![scsx_2023-02-24_14-01](https://user-images.githubusercontent.com/4263639/221185991-9c820e32-874c-42ae-8dd3-3e18e0f99ceb.png)

**Comment by Leon-Lj** [comment]
But of course I can't get the accessibility tree to stop complaining by editing the text area. Because that's no the part it's complaining about, it's upset about `ML__container`.

The best combination I've found is to have `aria-hidden=true` and `tabindex=0` on
```
markup += `<textarea class=ML__textarea__textarea autocapitalize=off autocomplete=off autocorrect=off spellcheck=false inputmode=none aria-hidden=true tabindex=0></textarea>`;
```
And then `aria-hidden=true`on
```
markup += '<span part=container class=ML__container aria-hidden=true><span part=content class=ML__content></span>';
```

This works (note that tabbing between elements doesn't work on the current master branch, so I don't know if changing the tab index has had a negative impact) and produces an a accessibility tree that looks almost clean.

![scsx_2023-02-24_16-03](https://user-images.githubusercontent.com/4263639/221211765-c1937b24-2837-4233-8e99-b7522d192a2e.png)

The last warning about keyboard can be solved by setting `role="textbox"` on the `<math-field>`. But doing so causes any `aria-label` or `aria-describedby` or such set on the `<math-field>` to not be read (at least not by NVDA). Oh the joy of working with screen readers. ;)

**Comment by Leon-Lj** [comment]
This issue has been solved in 0.89.4
